src/main/java/com/mathworks/ci/MatlabBuilderConstants.java

```diff
@@ -12,6 +12,7 @@
     static final double BASE_MATLAB_VERSION_EXPORTSTMRESULTS_SUPPORT = 9.6;
     
     static final String MATLAB_RUNNER_TARGET_FILE = "Builder.matlab.runner.target.file.name";
+    static final String MATLAB_TESTS_RUNNER_TARGET_FILE = "runMatlabTests.m";
     static final String MATLAB_RUNNER_RESOURCE = "com/mathworks/ci/MatlabBuilder/runMatlabTests.m";
```

Comment:
It seems like all of these variables contain values that repeat the `runMatlabTests` function name. You may consider refactoring down to 1 location where the `runMatlabTests` literal is written.

Reply:
Correct the underlying changes are in the actual copy method which is common across all builders . our implementation now is consistent for all new builder classes.   Variable MATLAB_RUNNER_TARGET_FILE is used in our old MatlabBuilder class as well which in near future will be deprecated.